src/sql/base/browser/ui/separator/separator.ts

```diff
@@ -0,0 +1,17 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { Disposable } from 'vs/base/common/lifecycle';
+
+export class Separator extends Disposable {
+	private readonly element: HTMLHRElement;
```

Comment:
`HTMLElement`

Reply:
I think HRElement might be more useful here, we might need to change specific stuff related to HRElement.

Is there a reason you're suggesting us to do Element instead?

Reply:
Just being consistent with all other components. All other components use `HTMLElement`

Reply:
Yeah I see what you mean but I would argue they should've also been scoped into the element they're creating.

Thanks!